internal/constellation/helm/BUILD.bazel

```diff
@@ -457,6 +457,7 @@ go_library(
         "charts/cert-manager/templates/controller-config.yaml",
         "charts/cert-manager/templates/poddisruptionbudget.yaml",
         "charts/cert-manager/templates/webhook-poddisruptionbudget.yaml",
+        "charts/edgeless/constellation-services/charts/autoscaler/templates/coredns-pdb.yaml",
```

Comment:
This appears to be out of order.

Reply:
I didn't edit this manually. The question is if we want to edit this manually or if we just leave this fully autogenerated.

Reply:
Well, if there is no buildifier step in our CI I and automation pastes stuff in this order, you are off the hook :)